components/41-organisms/collection/_collection.scss

```diff
@@ -7,13 +7,13 @@ ul.grid-5 {
   display: flex;
   flex-wrap: wrap;
   width: calc(100% + #{gutter()});
-  margin: 0 -#{gutter()/2} -1.2rem;
+  margin: 0 -#{gutter()*.5} -1.2rem;
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.